examples/linear_model.ipynb

```diff
@@ -0,0 +1,306 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `initialize()` function sets the states to their corresponding values. E.g., we have defined that our `x` value for our ThrownObject model to represent position in space, therefore, our initial value for the position in space would be the thrower_height parameter!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class ThrownObject(ThrownObject):\n",
+    "    def initialize(self, u=None, z=None):\n",
+    "        return self.StateContainer({\n",
+    "            'x': self.parameters['thrower_height'],\n",
+    "            'v': self.parameters['throwing_speed']\n",
+    "            })"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For our `threshold_met()`, we have defined the function to return a boolean function when our thrown object has met the initial state of 'impact', or in other words, has hit the ground. This occurs when our height is less than or equal to 0, and our speed is less than 0!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class ThrownObject(ThrownObject):\n",
+    "    def threshold_met(self, x):\n",
+    "        return {\n",
+    "            'falling': x['v'] < 0,\n",
+    "            'impact': x['x'] <= 0\n",
+    "        }"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, for our `event_state()`, we have defined our events to be when our object has hit the ground, or has made 'impact'. The `event_state()` function specifically returns the current state of our object at any given point of it's journey through the air!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class ThrownObject(ThrownObject):\n",
+    "    def event_state(self, x): \n",
+    "        x_max = x['x'] + np.square(x['v'])/(-self.parameters['g']*2)\n",
+    "        return {\n",
+    "            'falling': np.maximum(x['v']/self.parameters['throwing_speed'],0),\n",
+    "            'impact': np.maximum(x['x']/x_max,0) if x['v'] < 0 else 1\n",
+    "        }"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With these functions created, we can now run our ThrownObject Model!\n",
+    "\n",
+    "In this example, we will initialize our ThrownObject as `m`, and we'll define our `future_loading()` function (more information about future loading: https://nasa.github.io/progpy/prog_models_guide.html#future-loading) and our `simulate_to_threshold()` function (more information about simulation: https://nasa.github.io/progpy/prog_models_guide.html#simulation) to demonstrate the movement of the thrown object in air!"
```

Comment:
```suggestion
    "In this example, we will initialize our ThrownObject as `m`, and we'll define our `future_loading()` function (more information about future loading: https://nasa.github.io/progpy/prog_models_guide.html#future-loading). We'll use the `simulate_to_threshold()` function (more information about simulation: https://nasa.github.io/progpy/prog_models_guide.html#simulation) to simulate the movement of the thrown object in air."
```
When we originally developed this example, defining future_loading was necessary, but this is no longer the case. Should we take this out for simplicity/ease for reader?